tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/PublicAssetSetMethodContract.scala

```diff
@@ -2080,4 +2086,84 @@ trait PublicAssetSetMethodContract {
            |    }
            |}""".stripMargin)
   }
+
+  @Test
+  def createShouldSuccessWhenPublicAssetQuotaLimitIsExceededAndCleanUpSucceed(): Unit = {
```

Comment:
What happens when cleanup do not succeed?

Reply:
> createShouldReturnFailWhenPublicAssetQuotaLimitIsExceededAndCanNotCleanUp

Keep old behavior